osmosis/assetlist.json

```diff
@@ -1694,6 +1694,49 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/RAPTR.png"
         }
       ] 
+    },
+    {
+      "description": "Spitz on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
+          "exponent": 0,
+          "aliases": [
+            "spitz"
+          ]
+        },
+        {
+          "denom": "Spitz",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
+      "name": "SPITZ",
+      "display": "SPITZ",
+      "symbol": "SPITZ",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "chihuahua",
+            "base_denom": "factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz",
+            "channel_id": "channel-7"
+          },
+          "chain": {
+            "channel_id": "channel-113",
+            "path": "transfer/channel-113/factory/chihuahua17dgp6rf7wlhyaxth0j20h33zdv6nqat8xfezvy/spitz"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/spitz.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/spitz.png"
```

Comment:
Please do not upload 2 copies of the image. Please add image sync and point back to the Chihuahua image

Reply:
i modify the path by https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/spitz.png